locales/en/create-kafka-instance-with-sizes.json

```diff
@@ -0,0 +1,110 @@
+{
+  "all_region_unavailable_helper_text": "All regions in the selected cloud provider are temporarily unavailable. Select another cloud provider or try again later.",
```

Comment:
The original message I provided was too far out in the future, when we have more than one cloud provider. Until we have support for another cloud provider, we shouldn't tell the user to select another cloud provider. But there might also be future situations where we support more than one cloud provider, but don't provide an option.  So maybe we keep this and add another message, or maybe we just update this message for now.

In the case where only one cloud provider is supported, we should say this:
> All regions in the selected cloud provider are temporarily unavailable. Try again later.

Reply:
I'm taking back my suggestion for this message.

This helper text noted in my previous comment should only display for a use case where 2 or more providers are listed, and the selected provider has no available regions, but at least one region is available in another provider.

We should display the following message if there are no available regions for any provider. This message works whether we display one or more providers.
<img width="851" alt="image" src="https://user-images.githubusercontent.com/21063328/166305120-7b93f6cd-10f1-4a0b-aa6c-9b09e344783a.png">